### DISCUSSION TITLE
How do I make the words in the picture appear in the picture on the right？（plot_diff_cladogram,use R studio）

### DISCUSSION THREAD

**Discussion by Haoooo316** [ChiLiubio/microeco#363] [Q&A]
thank you for your attention to my problem. I really hope you can help me solve this problem
How do I make the words in the picture appear in the picture on the right？（plot_diff_cladogram,use R studio）

this is picture with problem:
[Rplot.pdf](https://github.com/ChiLiubio/microeco/files/15377246/Rplot.pdf)

this is my code:
> setwd("D:/0000/00/changdaojunqun/5 LEfSe")
> ASV<-read.table("ASV_table_with_taxonomy_4.2.0.txt",sep="\t",header = T,row.names=1,stringsAsFactors = T)
> GROUP<-read.table("sample_info_ori.txt",sep="\t",header = T,row.names=1,stringsAsFactors = T)
> TAX<-read.table("ASV_taxonomy_split.txt",sep="\t",header = T,row.names=1,stringsAsFactors = T)
> 
> GROUP$SampleID<-rownames(GROUP)
> TAX %<>% tidy_taxonomy
> df<-microtable$new(sample_table = GROUP,
>                    otu_table = ASV,
>                    tax_table = TAX,
>                    auto_tidy = F)
> 
> lefse<-trans_diff$new(dataset = df,
>                       method = "lefse",
>                       group = "Group",
>                       alpha=0.05,
>                       #taxa_level = "Species",
>                       p_adjust_method = "none"
>                       )
> lefse$plot_diff_cladogram(use_taxa_num =793 ,use_feature_num = 36,
>                           #clade_label_level =3,
>                           group_order = c("Control","Model","DY1L","DY1P"),
>                           color = c("Control"="#F0343D","Model"= "#F4A435" ,"DY1L"="#2B699E","DY1P"="#3AC82B"),
>                           #filter_taxa = 0.0001,
>                           #select_show_labels = use_labels,
>                           #only_select_show = F,
>                           #sep = "|",
>                           #branch_size = 0.5,
>                           alpha = 0.2,
>                           #clade_label_size = 0.1,
>                           node_size_offset = 1.5,
>                           #annotation_shape = 21,
>                           #annotation_shape_size = 4.5,
>                          )+
>       scale_fill_manual(values =c("#F0343D", "#F4A435" ,"#2B699E","#3AC82B"))+
>       guides(fill=guide_legend( ncol = 4, byrow = F),
>              #shape=guide_legend( ncol = 2, byrow = F)
>       )+
>   #geom_nodelab()+
>       theme(legend.position = "right", 
>             legend.box = "vertical",
>             legend.direction = "vertical",
>             #legend.title = element_blank(),
>             #legend.text = element_blank(),
>       )
>     ggsave("C.png",width= 130 , height= 140 , units="cm",dpi=400)

**Comment by Haoooo316** [comment]


**Reply by Haoooo316** [reply]
why my picture don't upload to the website?

**Comment by Haoooo316** [comment]
[Uploading Rplot.pdf…]()
[Uploading Rplot.pdf…]()

**Comment by ChiLiubio** [comment]
Please  try to save the picture with higher height, which can tighten the whole picture. Please also try to use other parameters (such as clade_label_size) to adjust the text size.
